modules/ROOT/pages/add-api-to-unified-schema.adoc

```diff
@@ -119,9 +129,29 @@ To edit the authentication method:
 . Select an authentication policy and complete any required fields.
 . Click *Save*.
 
+[[configure-mtls-for-an-api-added-to-the-unified-schema]]
+== Configure mTLS for an API Added to the Unified Schema
+
+If you need to configure mTLS for an API that you've added to the unified schema, you can do so on the *API details* page by adding a keystore. 
+
+You can also add a truststore to allow your API to be reached your unified schema.
+
+[NOTE]
+If the API already has a keystore or truststore, you can remove either and add new ones on the *API details* page. To do so, click *Remove and add a new one*.
+
+. Click *List of APIs added* and select the API that has the URL you want to edit.
+. Click *API details*.
+. Select *Configure Keystore* and provide the following:
```

Comment:
Does this sublist work? (I usually add the + and ** for a sublist.)

Reply:
they do work